PalasoUIWindowsForms/ImageToolbox/Cropping/ImageCropper.cs

```diff
@@ -358,33 +359,26 @@ public Image GetCroppedImage()
 
 			try
 			{
-				double z = 1.0/GetImageToCanvasScaleFactor();
-
-
-				int width = Math.Max(1, _rightGrip.Value - _leftGrip.Value);
-				int height = Math.Max(1, _bottomGrip.Value - _topGrip.Value);
-				var selection = new Rectangle((int) Math.Round(z*_leftGrip.Value),
-											  (int) Math.Round(z*_topGrip.Value),
-											  (int) Math.Round(z*width),
-											  (int) Math.Round(z*height));
-
 				//jpeg = b96b3c  *AE* -0728-11d3-9d7b-0000f81ef32e
 				//bitmap = b96b3c  *AA* -0728-11d3-9d7b-0000f81ef32e
 
-				Bitmap bmp = _originalImage as Bitmap;
-				if (bmp == null)
-					throw new ArgumentException("No valid bitmap");
-
-				CheckForInvalidImage();
-
 				//NB: this worked for tiff and png, but would crash with Out Of Memory for jpegs.
 				//This may be because I closed the stream? THe doc says you have to keep that stream open.
 				//Also, note that this method, too, lost our jpeg encoding:
 				//          return bmp.Clone(selection, _image.PixelFormat);
 				//So now, I first copy it, then clone with the bounds of our crop:
 
-				using (var copy = new Bitmap(_originalImage)) //**** here we lose the jpeg rawimageformat, if it's a jpeg. Grrr.
+				using (var copy = new Bitmap(_savedOriginalImage.Path)) //**** here we lose the jpeg rawimageformat, if it's a jpeg. Grrr.
```

Comment:
So the original code ended up copying the original twice, plus the output which might be nearly as big. Both versions have to make a bitmap of the whole original as well as the new bitmap. If the original was a very compressed jpg or sparse line drawing, we may still be using a lot of memory we don't need.
The top answer at http://stackoverflow.com/questions/734930/how-to-crop-an-image-using-c has another approach to copying part of an image into a new bitmap that might be worth considering.